crates/xilem_masonry/src/sequence.rs

```diff
@@ -254,38 +366,36 @@ impl<T, A> ViewSequence<T, A, ()> for () {
 impl<State, Action, M0, Seq0: ViewSequence<State, Action, M0>> ViewSequence<State, Action, (M0,)>
     for (Seq0,)
 {
-    fn build(&self, cx: &mut ViewCx, elements: &mut dyn ElementSplice) {
-        self.0.build(cx, elements);
+    type SeqState = Seq0::SeqState;
+    fn build(&self, cx: &mut ViewCx, elements: &mut dyn ElementSplice) -> Self::SeqState {
+        self.0.build(cx, elements)
     }
 
     fn rebuild(
         &self,
+        seq_state: &mut Self::SeqState,
         cx: &mut ViewCx,
         prev: &Self,
         elements: &mut dyn ElementSplice,
     ) -> ChangeFlags {
-        self.0.rebuild(cx, &prev.0, elements)
+        self.0.rebuild(seq_state, cx, &prev.0, elements)
     }
 
     fn message(
         &self,
+        seq_state: &mut Self::SeqState,
         id_path: &[ViewId],
         message: Box<dyn std::any::Any>,
         app_state: &mut State,
     ) -> MessageResult<Action> {
-        self.0.message(id_path, message, app_state)
+        self.0.message(seq_state, id_path, message, app_state)
     }
 
     fn count(&self) -> usize {
         self.0.count()
     }
 }
 
-const BASE_ID: NonZeroU64 = match NonZeroU64::new(1) {
```

Comment:
For future reference, TIL about [`NonZeroU64::MIN`](https://doc.rust-lang.org/std/num/type.NonZeroU64.html#associatedconstant.MIN)